pajbot/modules/quest.py

```diff
@@ -24,15 +24,15 @@ class QuestModule(BaseModule):
             type="options",
             required=True,
             default="say",
-            options=["say", "whisper", "me", "reply"],
+            options=["say", "whisper", "me", "announce", "reply"],
         ),
         ModuleSetting(
             key="action_tokens",
             label="MessageAction for !tokens",
             type="options",
             required=True,
             default="whisper",
-            options=["say", "whisper", "me", "reply"],
+            options=["say", "whisper", "me", "announce", "reply"],
```

Comment:
Neither this command
```suggestion
            options=["say", "whisper", "me", "reply"],
```